jsonrpc/websocket.go

```diff
@@ -72,13 +71,12 @@ func (ws *Websocket) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	var errClose websocket.CloseError
-	if errors.As(err, &errClose) {
-		ws.log.Infow("Client closed websocket connection", "status", websocket.CloseStatus(err))
+	if status := websocket.CloseStatus(err); status != 1 {
```

Comment:
should this be `!= -1` ?

https://github.com/nhooyr/websocket/blob/e2016acc42b29c1111d4e0b9971f27bfa98088ff/close.go#L84